lib/compiler/buildUitls.ts

```diff
@@ -1,3 +1,11 @@
+import { ImportFile } from '@resolver-engine/imports';
+
+export function buildSourcesObject(files: ImportFile[]): Record<string, { content: string}> {
```

Comment:
This function could actually be called by `buildInputObject` itself. This would simplify the calling code. Previously it had to be separate because they were differences between the compilers